src/System.Private.CoreLib/shared/System/Diagnostics/Tracing/StubEnvironment.cs

```diff
@@ -325,13 +325,13 @@ public static bool ReflectionOnly(this Assembly assm)
 }
 
 #if ES_BUILD_STANDALONE
-namespace Microsoft.Win32
+internal static partial class Interop
 {
     using System.Runtime.InteropServices;
     using System.Security;
 
     [SuppressUnmanagedCodeSecurityAttribute()]
```

Comment:
I think `SuppressUnmanagedCodeSecurityAttribute` can be removed.

Reply:
This code isn't used in the coreclr build; I'm not going to mess with it :)